src/config.js

```diff
@@ -1,3 +1,5 @@
 const PREDICTIONS_API_URL = 'http://100.26.159.108:5000';
 
-export { PREDICTIONS_API_URL };
\ No newline at end of file
+const HUNTING_SERVICE_URL = 'http://ec2-54-175-217-176.compute-1.amazonaws.com:8091/hunting/hunting/today';
```

Comment:
Hi there, can we keep EC2 address in the config file? (found in config.js) @lintongyu418 

Config file: https://github.com/mintso/pnnl-frontend-react/blob/main/src/config.js

You can find an example in importing the url here: https://github.com/mintso/pnnl-frontend-react/blob/main/src/DataAdaptor.js